app/bower.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this bower.json still required? Not in my eyes. We should remove it.

Reply:
bower.json removed.